tests/test_commands.py

```diff
@@ -345,6 +345,38 @@ def test_client_kill_filter_by_addr(self, r, r2):
         assert len(clients) == 1
         assert clients[0].get('name') == 'redis-py-c1'
 
+    @skip_if_server_version_lt('2.8.12')
+    def test_client_kill_filter_skipme_true(self, r):
+
+        r.client_setname('redis-py-c1')
+        clients = [client for client in r.client_list()]
+        clients_by_name = dict([(client.get('name'), client)
+                                for client in clients])
+        assert 'redis-py-c1' in clients_by_name
+
+        client_id = clients_by_name['redis-py-c1'].get('id')
+        resp = r.client_kill_filter(_id=client_id, skipme=True)
+        assert resp >= 0
+
+        clients = [client.get('name') for client in r.client_list()]
+        assert 'redis-py-c1' in clients
+
+    @skip_if_server_version_lt('2.8.12')
+    def test_client_kill_filter_skipme_false(self, r, r2):
+        r.client_setname('redis-py-c1')
+        clients = [client for client in r.client_list()]
+        clients_by_name = dict([(client.get('name'), client)
+                                for client in clients])
+        assert 'redis-py-c1' in clients_by_name
+
+        client_id = clients_by_name['redis-py-c1'].get('id')
+        resp = r.client_kill_filter(_id=client_id, skipme=False)
+        assert resp >= 1
+
+        # This client should have been killed!
+        with pytest.raises(exceptions.ConnectionError):
+            r.ping()
```

Comment:
Interesting thing: this test passes, but we leave around some error text in the test output:

```
  /redis-py/tests/test_commands.py:378: ResourceWarning: unclosed <socket.socket fd=5, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('172.18.0.7', 35596)>

    r.ping()
```